src/script/campaign/mod/vco-listeners.lua

```diff
@@ -51,6 +51,12 @@ end
 
 -- CHECKS --
 
+local function check_skv_mdr_number_upgrades_purchased(faction_key)
```

Comment:
Leave the vco there. Even if these are local functions, better to keep them consistent:
`check_vco_skv_mdr_number_upgrades_purchased(faction_key)`